xbmc/addons/addoninfo/AddonInfoBuilder.cpp

```diff
@@ -116,6 +117,12 @@ bool CAddonInfoBuilder::ParseXML(const AddonInfoPtr& addon, const TiXmlElement*
     return false;
   }
 
+  /*
+   * The function variable "repo" is only used when data stored on the internet
+   * are read out. A boolean value is then set here for easier identification.
+   */
+  const bool isRepoXMLContent = !repo.artdir.empty() && !repo.datadir.empty();
```

Comment:
isn't the `!repo.datadir.empty();` enough here? Or should we really check if the art dir is empty? art seems to have no relation with this

Reply:
Should be enough, there was just a little paranoia on my mind.